pom.xml

```diff
@@ -43,7 +43,7 @@ under the License.
   </contributors>
 
   <prerequisites>
-    <maven>${mavenVersion}</maven>
+    <maven>3.6.3</maven>
```

Comment:
We should change prerequisites in separate PR with separate issue to show it in release notes

Reply:
done